content/training/fullstack.mdx

```diff
@@ -100,7 +106,7 @@ trainingHeader:
   person: /images/piers.png
 _body:
   - trainingInformationItems:
-      - header: <span class="text-sswRed">Who</span> should apply?
+      - header: <span class="text-sswRed text-5xl">Who</span> should apply?
```

Comment:
Why was this added?

Reply:
I have added the "text-5xl" class because there is a desire to have the first letter in red and larger than the others.

Reply:
Who asked for this?

Reply:
It’s also changing the whole word, not the first letter

Reply:
Yes, I apologize, I meant "the first word" not "the first letter" on my comment.
You're correct Brady, I misread the sentence _"Increase font size for the "Who, What, When" section"_ (on the issue). I only enlarged the words, not the entire section.